src/sentry/incidents/endpoints/serializers.py

```diff
@@ -59,6 +59,41 @@
 CRITICAL_TRIGGER_LABEL = "critical"
 WARNING_TRIGGER_LABEL = "warning"
 
+# TODO: This is temporarily needed.
+# Use a function from discover to break the aggregate down into parts, and then compare the "field"
+# to a list of accepted fields, or a list of fields we need to translate.
+# This can be dropped once snuba can handle this aliasing.
+SUPPORTED_COLUMNS = [None, "tags[sentry:user]", "transaction.duration"]
+TRANSLATABLE_COLUMNS = {
+    "user": "tags[sentry:user]",
+    "dist": "tags[sentry:dist]",
+    "release": "tags[sentry:release]",
+}
+
+
+def forward_translate_snuba_field(aggregate):
+    field = resolve_field(aggregate)
+    column = field[1][0][1]
+    if column in SUPPORTED_COLUMNS:
+        return aggregate
+    elif column in TRANSLATABLE_COLUMNS.keys():
+        return aggregate.replace(column, TRANSLATABLE_COLUMNS[column])
+
+    return False
+
+
+def reverse_translate_snuba_field(aggregate):
+    field = resolve_field(aggregate)
+    column = field[1][0][1]
+    if column is None:
+        return aggregate
+
+    for field, translated_field in TRANSLATABLE_COLUMNS.items():
+        if translated_field == column:
+            return aggregate.replace(column, field)
+
+    return False
```

Comment:
We probably don't need the `return False` here. Could just return the unchanged aggregate, and then you could use it directly.